ledger/catchupaccessor.go

```diff
@@ -690,18 +691,22 @@ func (c *catchpointCatchupAccessorImpl) BuildMerkleTrie(ctx context.Context, pro
 					return
 				}
 				trie.SetCommitter(mc)
-				for _, accountHash := range hashesToWrite {
+				for _, hash := range hashesToWrite {
 					var added bool
-					added, err = trie.Add(accountHash)
+					added, err = trie.Add(hash)
 					if !added {
-						return fmt.Errorf("CatchpointCatchupAccessorImpl::BuildMerkleTrie: The provided catchpoint file contained the same account more than once. hash '%s'", hex.EncodeToString(accountHash))
+						return fmt.Errorf("CatchpointCatchupAccessorImpl::BuildMerkleTrie: The provided catchpoint file contained the same account more than once. hash '%s'", hex.EncodeToString(hash))
```

Comment:
Since this thing is all about debugging, let's improve it - how about telling us what kind of hash it is, by decoding`hash[4]`? We certainly all forgot that fact when debugging - we we presented with a hash that was a problem and didn't immediately look at it and say "Ahh, that's a kv hash".

Reply:
@jannotti Makes sense to me - https://github.com/algorand/go-algorand/pull/4808/commits/aeb4f191aad9c626e10f0c304feea98c0688b0fa#diff-da4e3bb740a328186b93cf1e154745e4f3bdf408866ae584c659b1408db689a9R698 attempts to enrich the error message.  Can you resolve if it matches your expectations?